tinygrad/renderer/cstyle.py

```diff
@@ -239,7 +241,8 @@ def render_kernel(self, function_name, kernel, bufs, uops, prefix=None):
   return {arg[3].name}2(c.thread_elements()[0], c.thread_elements()[1]);\n}}""")
     return super().render_kernel(function_name, kernel, bufs, uops, prefix)
 
-code_for_op_half = {BinaryOps.MAX: lambda a,b,dtype: f"__hmax({a},{b})" if dtype in (dtypes.half, dtypes.bfloat16) else f"max({a},{b})",
+code_for_op_half = {UnaryOps.RECIP: lambda x,dtype: f"({'(half)'if dtype == dtypes.half else ''}1/{x})",
```

Comment:
why override this?

Reply:
I remember to overrided this because some test_uops and test_ops where failing, I tests again without this override and all works (maybe I had some errors on my first try), submit the commit without this line now, thank you.